app/src/main/java/nextstep/jwp/http/request/RequestLine.java

```diff
@@ -0,0 +1,56 @@
+package nextstep.jwp.http.request;
+
+import nextstep.jwp.WebServer;
+import nextstep.jwp.http.common.HttpMethod;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RequestLine {
+
+    private HttpMethod httpMethod;
+    private URI uri;
+    private String versionOfProtocol;
```

Comment:
```suggestion
    private String protocolVersion;
```
이렇게 바꿔보는 건 어떨까요? 개인 취향이니 넘어가도 됩니다!